frontend/src/components/pages/Default.tsx

```diff
@@ -37,6 +38,21 @@ const TeamInfoDisplay = () => {
   );
 };
 
+const DesignLibraryExamples = () => {
+  const { teamName, numTerms, members, isActive } = useContext(SampleContext);
+  return (
+    <div>
+      {/* Example: how to apply custom colors */}
+      <ButtonGroup gap='4'>
+          <ChakraButton bg="blue.50" color="white"> Test Chakra Button </ChakraButton>
```

Comment:
I'm just being unnecessarily picky, but maybe we can write the specific colour description instead of "Test Chakra Button" (i.e. "blue", "medium blue", "light blue", "very light blue")

Reply:
no, love it! good catch